docs/methods/cardanoGetAddress.md

```diff
@@ -163,104 +390,23 @@ const result = await TrezorConnect.cardanoGetAddress({
     },
     protocolMagic: 764824073,
     networkId: 0,
-    address: "Ae2tdPwUPEZ5YUb8sM3eS8JqKgrRLzhiu71crfuH2MFtqaYr5ACNRdsswsZ",
+    address: "Ae2tdPwUPEZ7XH2vpa1gDzioNKAWEHWU49AREQg2pD9wr3fwXHPgKbyNwkZ",
 });
 // dont forget to hide your custom UI after you get the result!
 ```
-
-### Result
-Result with only one address
+#### Response
 ```javascript
 {
     success: true,
     payload: {
         addressParameters: {
-            addressType: number,
-            path: Array<number>, // hardend path
-            stakingPath?: Array<number>, // hardend path
-            stakingKeyHash?: string,
-            certificatePointer?: {
-                blockIndex: number,
-                txIndex: number,
-                certificatePointer: number,
-            }
+            addressType: 8,
+            path: [(44 | 0x80000000) >>> 0, (1815 | 0x80000000) >>> 0, (0 | 0x80000000) >>> 0, 0, 0]
```

Comment:
I'm not sure if actual examples are actually that beneficial for understanding and it does not seem to align with other docs in the folder which also reference types. Possibly for addresses or other non-trivial structures it may make sense to leave a comment with a specific example but otherwise I don't see that much value in providing examples of paths if it is a general trezor concept